cpp/src/bitmask/null_mask.cu

```diff
@@ -81,31 +81,31 @@ namespace {
 
 /**---------------------------------------------------------------------------*
  * @brief Counts the number of non-zero bits in a bitmask in the range
- * `[start_bit_index, stop_bit_index]`.
+ * `[first_bit_index, last_bit_index]`.
  *
- * Expects `0 <= start_bit_index < stop_bit_index`.
+ * Expects `0 <= first_bit_index < last_bit_index`.
```

Comment:
Shouldn't this be <= since it's inclusive? Otherwise you can't copy a single bit.

```suggestion
* Expects `0 <= first_bit_index <= last_bit_index`.
```